unleash.xcodeproj/project.pbxproj

```diff
@@ -846,10 +831,12 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.silvercar.Unleash;
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
-				SDKROOT = iphoneos;
+				SDKROOT = macosx;
```

Comment:
it's hard to tell from the diff but do these all effect simulator builds?

Reply:
Not at all, from what I tested. The main thing is that the library can be used to support different client types, if we choose.

Reply:
nice